frontend/src/app/shared/components/calendar/calendar.component.ts

```diff
@@ -1,14 +1,57 @@
-import { Component, Input } from '@angular/core';
+import { Component, forwardRef, Input } from '@angular/core';
+import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
 
 @Component({
     selector: 'app-calendar',
     templateUrl: './calendar.component.html',
     styleUrls: ['./calendar.component.sass'],
+    providers: [
+        {
+            provide: NG_VALUE_ACCESSOR,
+            // eslint-disable-next-line no-use-before-define
```

Comment:
delete this

Reply:
without this, it can't be compiled on server. I saw in code that we sometimes disable `eslint` if can't fix it

Reply:
Find the way to fix this without disabling